source/clear-linux/get-started/bare-metal-install/bare-metal-manual-install.rst

```diff
@@ -0,0 +1,284 @@
+.. _bare-metal-manual-install:
+
+Bare metal manual installation guide
+####################################
+
+This section is a continuation of steps from the :ref:`bare-metal-install`
+page.
+
+Selecting the :guilabel:`< Manual(Advanced) >` menu option as seen in figure
+1, you can do the following additional tasks during the |CL| Installer setup:

```

Comment:
With the imperative you can simplify this line quite a bit:
to do the following tasks during the |CL| installation:

Reply:
done